index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en"> 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100;200&display=swap" rel="stylesheet">
+</head>
+<body>
```

Comment:
- [ ] Kindly fix the issue with the design. Some elements are overlapping on each other and other elements are hiding out of the screen. Kindly try to follow the design on Figma as much as possible, this includes `font-size`, font `color`, `margin` and `padding` 💯 

## Your design:
![Animation](https://user-images.githubusercontent.com/50721479/192114560-b508f77e-275e-4715-9422-dcbee6ec2d64.gif)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192114565-4fee73c8-1b21-40f3-8795-f29924321f8d.png)

Reply:
Fixed.